openscm/timeseries.py

```diff
@@ -150,34 +157,38 @@ def _calc_linearization_values(values: np.ndarray) -> np.ndarray:
     )
 
 
-def _calc_linearization(
-    values: np.ndarray, timeframe: Timeframe
+def _calc_continuous(
```

Comment:
We should rename this function (though it's going to change in #126 anyway). `_calc_stepwise_linear_function`?

Reply:
The way it's implemented now it's not calculating a stepwise linear function, it's calculating a continuous linear interpolation. I think a name which makes it clear that we're calculating a continuous function here would be useful (and then maybe we add args for how to make things continuous that can take arguments like 'linear' or 'cubic' and 'integral_preserving' or not).